pkg/bundle/internal/target/target.go

```diff
@@ -442,6 +446,14 @@ func TrustBundleHash(data []byte, additionalFormats *trustapi.AdditionalFormats)
 		_, _ = hash.Write([]byte(*additionalFormats.PKCS12.Password))
 	}
 
+	// Add Target annotations and labels to the hash so it becomes aware of changes and triggers an update.
+	for targetAnnotationKey, targetAnnotationValue := range target.GetAnnotations() {
+		_, _ = hash.Write([]byte(targetAnnotationKey + targetAnnotationValue))
+	}
+	for targetLabelKey, targetLabelValue := range target.GetLabels() {
+		_, _ = hash.Write([]byte(targetLabelKey + targetLabelValue))
+	}
```

Comment:
```suggestion
	for k, v := range target.GetAnnotations() {
		_, _ = hash.Write([]byte(k + v))
	}
	for k, v := range target.GetLabels() {
		_, _ = hash.Write([]byte(k + v))
	}
```

More nitting, but it makes it a bit more compact, as the scope is so small.